roles/master/tasks/main.yml

```diff
@@ -10,6 +10,17 @@
   tags:
     - master
 
+- name: install kubernetes node
```

Comment:
Good point. Btw we can install meta-package `kubernetes` instead of separate packages.